src/apps/saddle/evmos/saddle.balance-fetcher.ts

```diff
@@ -0,0 +1,90 @@
+import { Inject } from '@nestjs/common';
+
+import { drillBalance } from '~app-toolkit';
+import { APP_TOOLKIT, IAppToolkit } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { presentBalanceFetcherResponse } from '~app-toolkit/helpers/presentation/balance-fetcher-response.present';
+import { BalanceFetcher } from '~balance/balance-fetcher.interface';
+import { MetaType } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { SaddleCommunalFarm, SaddleContractFactory, SaddleMiniChefV2 } from '../contracts';
+import { SADDLE_DEFINITION } from '../saddle.definition';
+
+@Register.BalanceFetcher(SADDLE_DEFINITION.id, Network.EVMOS_MAINNET)
+export class EvmosSaddleBalanceFetcher implements BalanceFetcher {
+  constructor(
+    @Inject(APP_TOOLKIT) private readonly appToolkit: IAppToolkit,
+    @Inject(SaddleContractFactory) private readonly contractFactory: SaddleContractFactory,
+  ) {}
+
+  private async getPoolTokenBalances(address: string) {
+    return this.appToolkit.helpers.tokenBalanceHelper.getTokenBalances({
+      address,
+      appId: SADDLE_DEFINITION.id,
+      groupId: SADDLE_DEFINITION.groups.pool.id,
+      network: Network.EVMOS_MAINNET,
+    });
+  }
+
+  private async getSaddleMiniChefV2(address: string) {
+    return this.appToolkit.helpers.singleStakingContractPositionBalanceHelper.getBalances<SaddleCommunalFarm>({
+      address,
+      appId: SADDLE_DEFINITION.id,
+      groupId: SADDLE_DEFINITION.groups.communalFarm.id,
+      network: Network.EVMOS_MAINNET,
+      resolveContract: ({ address, network }) => this.contractFactory.saddleCommunalFarm({ address, network }),
+      resolveStakedTokenBalance: ({ contract, address, multicall }) =>
+        multicall.wrap(contract).lockedLiquidityOf(address),
+      resolveRewardTokenBalances: ({ contract, address, multicall }) => multicall.wrap(contract).earned(address),
+    });
+  }
```

Comment:
Delete lines 30-41, not needed anymore since you removed the communal farm group.

Reply:
Ok - completed.